NEST-279: Migrate Lombiq.Privacy to Orchard Core and add it to DotNest Core (#1)
Lombiq/Orchard-Privacy: Readme.md

**Important!** Using this module will not make your site GDPR-compliant alone.

Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!

> The module is also available for [DotNest](https://dotnest.com/) sites.

Reviewer: ```suggestion
It's also available on all sites of [DotNest, the Orchard SaaS](https://dotnest.com/).
```

**Important!** Using this module will not make your site GDPR-compliant alone.

Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!

The module is also available for [DotNest](https://dotnest.com/) sites.

**NOTE:** This module has an Orchard 1 version in the [dev-orchard-1 branch](https://github.com/Lombiq/Orchard-Privacy/tree/dev-orchard-1).

## Features

The module consists of the following independent features:

### Consent Banner Feature

Shows a banner where users can accept the privacy policy. 

If the user is unauthenticated, they can use the banner to accept or reject the privacy statement, their decision will be stored in a cookie by the browser. 

> If the user is authenticated, they must accept the privacy policy via the banner. The fact of acceptance will be stored in the `PrivacyConsent` section of the user's properties. 

Reviewer: Nem igazán világos, hogy mi ez a "must".

Author: Amit itt is irtam: https://github.com/Lombiq/Orchard-Privacy/pull/1#discussion_r717570804.
Jelenleg authentikált usernek el kell fogadnia a feltételeket az esetleges regcheckbox bekapcsolás miatt. De ezt mostmár hogy a logika egy helyre van szervezve könnyebb módosítani(legalábbis mondom most aztán ki tudja)

Reviewer: A logika jó, csak a dokumentációval van problémám. Én értem :), csak itt másoknak is ki kellene fejteni, hogy mire gondolsz, mi a "must" és miért. Tisztább lenne kb. "If the user is authenticated, their only option is to accept the privacy policy via the banner. This is so because it is assumed that during registration they already accepted a suitable privacy policy, it's just that the Lombiq.Privacy or something similar wasn't used at the time."

The module is also available for [DotNest](https://dotnest.com/) sites.

> **NOTE:** This module has an Orchard 1 version in the [dev-orchard-1 branch](https://github.com/Lombiq/Orchard-Privacy/tree/dev-orchard-1).

Reviewer: A Readme címe legyen inkább "Lombiq Privacy for Orchard Core". Lásd: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution

Reviewer: Add hozzá az OSOCE-hez is, egy ottani PR-ral, és a Readme második bekezdésével: https://github.com/Lombiq/Open-source-template

Author: Itt még megjegyzem hogy a [dev-orchard-1 branch](https://github.com/Lombiq/Orchard-Privacy/tree/dev-orchard-1) Readme-jében is be kell linkelnem a core-os branchet.

> **NOTE:** This module has an Orchard 1 version in the [dev-orchard-1 branch](https://github.com/Lombiq/Orchard-Privacy/tree/dev-orchard-1).
> 

Reviewer: ```suggestion
**NOTE:** This module has an Orchard 1 version in the [dev-orchard-1 branch](https://github.com/Lombiq/Orchard-Privacy/tree/dev-orchard-1).

```

## Features

The module consists of the following independent features:

### Consent Banner Feature

Shows a banner where unauthenticated users can accept the privacy policy. 

### Registration Consent Feature

Shows a privacy consent checkbox on the registration form that needs to be checked by the users to be able to register. After registration, the user's consent is stored in the Privacy section of the user's properties.

**NOTE:** If the user registered before this feature was enabled then they can accept the consent with the consent banner (if it's enabled). The consent will be stored in this case as well. 

### Form Consent Feature

> Adds a new `ConsentCheckbox` content type that can be attached to Dynamic Forms. In this case the users must accept the privacy policy before they can post content to the site. You can validate the consent wiht `Validate Consent Checkbox` workflow activity on the following way:

Reviewer: ```suggestion
Adds a new Consent Checkbox widget content type that can be added to forms with the Forms module. In this case, users must accept the privacy policy before they can post content to the site. You can validate the consent with the `Validate Consent Checkbox` workflow activity in the following way:
```

> ![Consent Checkbox Workflow](Docs/ConsentCheckboxWorkflow.png)

Reviewer: Apróság, de a fájl nevében kicsivel legyen a png (emiatt nem látszik itt: https://github.com/Lombiq/Orchard-Privacy/blob/9a19dccb194028f1003dc8f194b8fb937b7db7de/Readme.md).

If the user is unauthenticated, they can use the banner to accept or reject the privacy statement, their decision will be stored in a cookie by the browser. 

> If the user is authenticated, their only option is to accept the privacy policy via the banner. This is so because it is assumed that during registration they already accepted a suitable privacy policy, it's just that the Lombiq.Privacy or something similar wasn't used at the time.

Reviewer: Már bennem sem lehet bízni:
```suggestion
If the user is authenticated, their only option is to accept the privacy policy via the banner. This is so because it is assumed that during registration they already accepted a suitable privacy policy, it's just that the Lombiq.Privacy module or something similar wasn't used at the time.
```
